app/views/layouts/application.html.slim

```diff
@@ -1,26 +1,29 @@
 doctype html
-html
+html(lang='en')
   head
+    meta(charset='utf-8')
+    meta(http-equiv='X-UA-Compatible', content='IE=edge')
+    meta(name='viewport', content='width=device-width, initial-scale=1')
+    // The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags
+    meta(name='description', content='')
+    meta(name='author', content='')
+    link(rel='icon', href='/favicon.ico')
     title Velum
     = stylesheet_link_tag 'application', media: 'all'
     = javascript_include_tag 'application'
```

Comment:
this is a 404. See my other comment above (app/assets/javascripts/application.js)